src/ray/protobuf/gcs_service.proto

```diff
@@ -23,15 +23,15 @@ message AddJobRequest {
 }
 
 message AddJobReply {
-  bool success = 1;
+  string status = 1;
```

Comment:
What about just define a `Status` message in probobuf, which has 2 fields `code` and `message`?
Then we don't need that `STATUS_SEPARATOR`.

Reply:
fixed